From 229441f111066fb4340ac6ae49cf26db2f81b884 Mon Sep 17 00:00:00 2001
From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>
Date: Fri, 3 Nov 2017 13:03:42 +0100
Subject: [PATCH 5/9] machine compat: pci_bridge/shpc always enable

RH-Author: Dr. David Alan Gilbert <dgilbert@redhat.com>
Message-id: <20171103130342.9839-1-dgilbert@redhat.com>
Patchwork-id: 77494
O-Subject: [RHEL-7.5 qemu-kvm-rhev PATCH 1/1] machine compat: pci_bridge/shpc always enable
Bugzilla: 1508271
RH-Acked-by: Marcel Apfelbaum <marcel@redhat.com>
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>

The 'shpc' property on bridges has changed its default back and
forward.
Upstream it's off on 2.9 machine types but on for all the others
(upstream dc0ae767 turned it off, 2fa35662 reverted it).
Downstream we always had it on, even for 7.4 that was derived from
2.9.
Unfortunately we imported the 2.9 entry that turned it off, which
breaks migration.  Remove that entry, shpc is now on again for
all our machine types.

Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/compat.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/include/hw/compat.h b/include/hw/compat.h
index 85c6cbe..7a2a4a6 100644
--- a/include/hw/compat.h
+++ b/include/hw/compat.h
@@ -419,10 +419,6 @@
  */
 #define HW_COMPAT_RHEL7_4 \
     { /* HW_COMPAT_RHEL7_4 */ \
-        .driver   = "pci-bridge",\
-        .property = "shpc",\
-        .value    = "off",\
-    },{ /* HW_COMPAT_RHEL7_4 */ \
         .driver   = "intel-iommu",\
         .property = "pt",\
         .value    = "off",\
-- 
1.8.3.1

